src/components/Counter.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, Fragment, useEffect } from 'react';
+
+import { db } from '../lib/firebase';
+
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+
+const Counter = () => {
+  const [counter, setCounter] = useState(
+    JSON.parse(localStorage.getItem('counter')) || 1,
```

Comment:
Hi @YuriDevAT,  localStorage functionality is  here for a different reason - to avoid the counter reset on a page refresh. Documents are added to a collection with the help of the add method :
```
 db.collection('counters').add({
      value: counter,
    });
```

Reply:
Just keep in mind that adding this functionality could become obsolete already for your next issue. For the demos, you only have to present an unstyled component which increments in the app and also in firestore and vice versa (it doesn't matter in which order, or if there will be a page refresh)

Reply:
Completely agree, it wasn't necessary. But the other team wanted a sorted list of numbers to be rendered, and it's why I had to add sorting + local storage functionality on top.

Reply:
hi @YuriDevAT, local storage is removed. Bear in mind, though, now that  the click count is no longer retained,  the sequence of numbers that will get output to the page will be broken.